app/assets/stylesheets/chats.scss

```diff
@@ -0,0 +1,29 @@
+.chat-table {
+  width: 350px;
+  margin: 0 auto;
+}
+
+.chat-table th, .chat-table td {
+    border-bottom: 1px solid #ddd;
+}
+
+.chat-receiver {
+  cursor: pointer;
+}
+
+.chat-topic {
+  margin-left: 228px;
+}
+
+.chat-table tr:hover {
+  background-color: #FFFFE0;
+}
+
+.messages {
+  overflow: auto;
+  height:700px
```

Comment:
Take care of spaces here. It's really important cause its add readability to the code. Always leave one blank space after the key.

```
{
  key: value
}
```
That way it will be easier to read next time you have to go through the code 😉